src/pages/Home.jsx

```diff
@@ -81,37 +83,39 @@ const DocumentsSelectionTopBar = () => {
   </div>
 }
 
-const DocumentFileText = ({doc}) => {
+const DocumentFileText = ({docInfo}) => {
   return <div className="flex flex-col h-14">
     <div className="flex-1 text-gray-600">
-      Doc Title {doc}
+      {docInfo.name}
     </div>
     <div className="w-full text-right text-gray-400 text-xs">
-      Date
+      {Moment(docInfo.dateLastModified).format('MMM D, YYYY')}
```

Comment:
what format is `MMM D`?

Reply:
Ex: Jan 3, 2001

Reply:
ooo I see ok ty, does that handle double digit dates like Jan 17? if yes then all g

Reply:
Ya